en/docs/develop/hosting-authentication-endpoint-on-a-different-server.md

```diff
@@ -24,11 +24,7 @@ Server).
     First, get a copy of
     `         <IS_HOME>/repository/deployment/server/webapps        `
     `         /authenticationendpoin.war        ` to
-    `         <WebApp_HOME>/        ` and unzip it. Make sure to get the
-    `         authenticationendpoin.war        ` after the WUM update and
-    NOT the extracted `         authentication        `
-    `         endpoint        ` in
-    `         <IS_HOME>/repository/deployment/server/webapps/        `
```

Comment:
We may need this as well since this is speaking generally of what we should do when a WUM update is taken for authenticationendpoin.war.

Reply:
fixed 90f9296684e786c30e9e1b00e607e708254f28b5